src/prpy/util.py

```diff
@@ -725,6 +725,58 @@ def GeodesicDistance(t1, t2, r=1.0):
     error[3] = r*error[3]
     return numpy.linalg.norm(error)
 
+
+def GetGeodesicDistanceBetweenTransforms(T0, T1):
+    """
+    Wrapper, to match GetGeodesicDistanceBetweenQuaternions()
+    """
+    return GeodesicDistance(T0, T1, r=1.0)
+
+
+def GetEuclideanDistanceBetweenTransforms(T0, T1):
+    """
+    Calculate the Euclidean distance between two 4x4 transforms.
+    (also called L2 or Pythagorean distance)
```

Comment:
This appears to only consider *translational* distance. This is not clear from the name of the function of the `__docstring__`.